webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/RuleApi.scala

```diff
@@ -454,7 +454,8 @@ class RuleApi(
 
     def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
       implicit val qc: QueryContext = authzToken.qc
-      serviceV14.getCategoryTree().toLiftResponseOne(params, schema, s => Some(s.ruleCategories.id))
+      val ruleIdsFilter = req.params.get("ruleId").map(_.flatMap(RuleId.parse(_).toOption))
```

Comment:
Can you add a comment explaining what that param does, the expected format ? And update the corresponding API doc?

Reply:
It's likely that we need to increment the API version number too, since the bevavior won't be the same

Reply:
(I think we said it's ok in that case, because we don't add a public api endpoint and we don't change the behavior of existing calls)

Reply:
Sure ! Here is the commit : https://github.com/Normation/rudder/pull/5297/commits/24d8d1059456a8cbe4b8bad622655b5af054cb8e

Reply:
+ another one, I copied from an existing API doc but the format should be `?ruleId=id1&ruleId=id2...`, `exploded` should be `true`

Reply:
we don't use that format anywhere, you should use `ruleIds=id1,id2...`

Reply:
Okay, I also found that `rules` is more consistent than `ruleIds` from other endpoints, so I changed the key : https://github.com/Normation/rudder/pull/5297/commits/b9e129d3eda2081bc6c3521b83666fda2fab98a3

Reply:
As we discussed, I created another private endpoint instead of changing this public one. It should return only needed info for rules related to a given group : https://github.com/Normation/rudder/pull/5297/commits